ledger-core/virtual/execute/execute.go

```diff
@@ -241,7 +241,6 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 		case object.Unknown:
 			panic(throw.Impossible())
 		case object.Inactive:
-			// attempt to create object that is deactivated :(
 			panic(throw.Impossible())
```

Comment:
this is very possible situation

Reply:
example?

Reply:
duplicate request of constructor after deactivation in previous pulse. I wrote about that in slack.

Reply:
move this check to stepStartRequestProcessing